kernel/vmbytegen.ml

```diff
@@ -658,18 +658,6 @@ let rec compile_lam env cenv lam sz cont =
       compile_fv cenv fv.fv_rev sz
         (Kclosurecofix(fv.size, init, lbl_types, lbl_bodies) :: cont)
 
-  | Lif(t, bt, bf) ->
```

Comment:
If we want to merge the native and the VM ASTs, it might be better to keep this dead code.

Reply:
If I am not mistaken, no code of `nativelambda.ml` can create `Lif` constructs. In other words, even after merging both native and bytecode ASTs, this part of the code would still stay dead. (This does not mean that the corresponding code in `nativecode.ml` is dead too. Indeed, contrary to `Lif`, the `MLif` construct can be generated, but that is specific to the native backend.)